### PR TITLE
RBAC support added to CLI Migrate

### PR DESCRIPTION
**Pull request by huseyinbabal**
## Description

Changes proposed in this pull request:

- RBAC section is now shipped to Cloud on migrate command

## Testing

- Run cloud app locally
- Deploy botkube in local k8s cluster
- Go to cli folder `cd cmd/cli`
- Build CLI `go build -o bctl main.go`
- Login to local cloud `./bctl login --cloud-dashboard-url http://localhost:3000`
- Migrate deployment `./bctl migrate --cloud-api-url http://localhost:8080/graphql`

## Related issue(s)

Relates https://github.com/kubeshop/botkube-cloud/issues/99

### CODE REVIEW

**Review by mszostok — APPROVED** [review]
LGTM

I spot some problems but we will address them in the follow-ups 👍

**Comment by mszostok** [comment]
@huseyinbabal Please also update this document https://docs.botkube.io/cli/migrating-installation-to-botkube-cloud

also as agreed on Slack, please describe that the `extraObjects` are not migrated. So the `helm upgrade` that we perform simply removes all of them, so if user had cluster roles or bindings there, then they need to migrate them on their own to make sure that migrated plugin RBAC config will work properly.

File: internal/ptr/ptr.go [hunk 1]
```diff
@@ -14,6 +14,15 @@ func ToSlice[T any](in []*T) []T {
...
 }
 
+// FromSlice returns slice of pointers for a given type.
+func FromSlice[T any](in []T) []*T {
+	out := make([]*T, 0, len(in))
+	for idx := range in {
+		out = append(out, &in[idx])
+	}
+	return out
+}
```
[the remark this answers is not in the document]
Comment (huseyinbabal) [reply internal/ptr/ptr.go]: This function is not needed anymore

**Comment by huseyinbabal** [comment]
> @huseyinbabal Please also update this document https://docs.botkube.io/cli/migrating-installation-to-botkube-cloud
> 
> also as agreed on Slack, please describe that the `extraObjects` are not migrated. So the `helm upgrade` that we perform simply removes all of them, so if user had cluster roles or bindings there, then they need to migrate them on their own to make sure that migrated plugin RBAC config will work properly.

https://github.com/kubeshop/botkube-docs/pull/254

**Pull request merged**